<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_equipement</class>
 <widget class="QMainWindow" name="interface_equipement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Equipement">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1291</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Ajout">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>341</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;</string>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>43</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/*font: 8pt &quot;MS Shell Dlg 2&quot;;*/
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Id_equipement :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="idaj">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>43</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(247, 247, 247);
/*color: rgb(248, 248, 248);*/
font: 75 14pt &quot;Source Code Pro&quot;;
border-radius:9px;
text-align: center;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>107</y>
         <width>110</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nom_centre :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>169</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nom_equipement :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>235</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Dernier_date_intervention :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>27</x>
         <y>303</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Etat_equipement :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nomcaj">
       <property name="geometry">
        <rect>
         <x>186</x>
         <y>108</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(247, 247, 247);
font: 75 14pt &quot;Source Code Pro&quot;;
border-radius:9px;
text-align: center;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nomeaj">
       <property name="geometry">
        <rect>
         <x>196</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(247, 247, 247);
font: 75 14pt &quot;Source Code Pro&quot;;
border-radius:9px;
text-align: center;</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="date">
       <property name="geometry">
        <rect>
         <x>218</x>
         <y>234</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="etataj">
       <property name="geometry">
        <rect>
         <x>197</x>
         <y>304</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(247, 247, 247);
font: 75 14pt &quot;Source Code Pro&quot;;
border-radius:9px;
text-align: center;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton

{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(78, 205, 196);
	color: #fff;
	border-top-right-radius: 15px;
	border-top-left-radius: 15px;
	border-bottom-right-radius: 15px;
	border-bottom-left-radius: 15px;
	padding: 5px;
}



QPushButton::disabled
{
	
background-color: rgb(9, 55, 76);

}


QPushButton::hover
{
background-color: rgb(9, 55, 76);


}


QPushButton::pressed
{
background-color: rgb(9, 55, 76);


}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>212</x>
         <y>390</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton

{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(78, 205, 196);
	color: #fff;
	border-top-right-radius: 15px;
	border-top-left-radius: 15px;
	border-bottom-right-radius: 15px;
	border-bottom-left-radius: 15px;
	padding: 5px;
}



QPushButton::disabled
{
	
background-color: rgb(9, 55, 76);

}


QPushButton::hover
{
background-color: rgb(9, 55, 76);


}


QPushButton::pressed
{
background-color: rgb(9, 55, 76);


}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>260</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Chercher :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditchercher">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>260</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1082</x>
        <y>260</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton

{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(78, 205, 196);
	color: #fff;
	border-top-right-radius: 15px;
	border-top-left-radius: 15px;
	border-bottom-right-radius: 15px;
	border-bottom-left-radius: 15px;
	padding: 5px;
}



QPushButton::disabled
{
	
background-color: rgb(9, 55, 76);

}


QPushButton::hover
{
background-color: rgb(9, 55, 76);


}


QPushButton::pressed
{
background-color: rgb(9, 55, 76);


}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>-230</x>
        <y>-120</y>
        <width>1541</width>
        <height>781</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/res/bg_equipement.jpg);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;
font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Gestion Equipements</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewe">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>50</y>
        <width>611</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="stat4">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>316</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>groupBox</zorder>
     <zorder>label_8</zorder>
     <zorder>lineEditchercher</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>label_11</zorder>
     <zorder>tableViewe</zorder>
     <zorder>stat4</zorder>
    </widget>
    <widget class="QWidget" name="Metier">
     <attribute name="title">
      <string>Autre</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>E-mail :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>481</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton

{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(78, 205, 196);
	color: #fff;
	border-top-right-radius: 15px;
	border-top-left-radius: 15px;
	border-bottom-right-radius: 15px;
	border-bottom-left-radius: 15px;
	padding: 5px;
}



QPushButton::disabled
{
	
background-color: rgb(9, 55, 76);

}


QPushButton::hover
{
background-color: rgb(9, 55, 76);


}


QPushButton::pressed
{
background-color: rgb(9, 55, 76);


}</string>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>-170</x>
        <y>-120</y>
        <width>1461</width>
        <height>771</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/res/bg_equipement.jpg);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Trier selon :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>60</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:9px;
</string>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton

{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(78, 205, 196);
	color: #fff;
	border-top-right-radius: 15px;
	border-top-left-radius: 15px;
	border-bottom-right-radius: 15px;
	border-bottom-left-radius: 15px;
	padding: 5px;
}



QPushButton::disabled
{
	
background-color: rgb(9, 55, 76);

}


QPushButton::hover
{
background-color: rgb(9, 55, 76);


}


QPushButton::pressed
{
background-color: rgb(9, 55, 76);


}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxox">
       <property name="geometry">
        <rect>
         <x>67</x>
         <y>60</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>365</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom_equipement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom_centre</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>361</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>label_9</zorder>
     <zorder>textBrowser_2</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>label_6</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>comboBox_2</zorder>
     <zorder>tableView</zorder>
     <zorder>label_7</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
